src/applications/mhv/secure-messaging/tests/containers/FolderListView.unit.spec.jsx

```diff
@@ -1,62 +1,70 @@
-import React from 'react';
-import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
-import { expect } from 'chai';
-import messages from '../fixtures/search-response.json';
-import folder from '../fixtures/folder-inbox-metadata.json';
-import reducer from '../../reducers';
-import FolderListView from '../../containers/FolderListView';
+/**
+ * @Note : This component is currently not used in SM application.
+ *         Consider deleting this file if there are no plans to use
+ *         this component in the future, or refactor this file to test
+ *         for the 'FilterBox' component.
+ */

```

Comment:
I agree, we can get rid of FolderListView component and this unit test. @robertbylight should have had test coverage done for FilterBox on his branch which is yet to be merged.